### PR TITLE
BWDB-10879

### PR DESCRIPTION
**Pull request by jhough07**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Adding additional Import Campaign notes to common issues experienced  by customers so far

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

File: campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md [hunk 1]
```diff
@@ -12,8 +12,15 @@ This walks through how to programmatically import and view your campaigns _via A
...
 * Your account has 10dlcImportCampaigns product feature enabled
 * Your account has your CSP ID associated to it
+* Your API User has the Campaign Management role assigned
 * You have provisioned campaigns with The Campaign Registry (TCR) through your Campaign Service Provider (CSP) 
 
+## Important Notes
+
+* If you get a '403 Unauthorized' error response while making an HTTP Request to any of the Import endpoints, 
+you will need to reach out to the Implementation team to get the Campaign Management role assigned to your API User.
+* TCR currently has rate limits set on their HTTP REST endpoints. During high volume events, it is possible our APIs will be rate limited and respond with '429 Too Many Requests'.
```
Comment (jmulford-bw) [inline campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md:22]:
```suggestion
* TCR currently has rate limits set on their HTTP REST endpoints. If you are sending a lot of traffic, it is possible that you will hit a rate limit and the API will respond with '429 Too Many Requests'.
```

Comment (jmulford-bw) [reply campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md:22]: The original sentence reads like the entire API is rate limited across all users. I'm guessing it's rate limited per user?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]
1 small change but otherwise looks good

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-bwdb-10879.s3-website-us-east-1.amazonaws.com/index.html

File: campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md (hunk 1 above)
Comment (jhough07) [reply campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md:22]: Unfortunately it is across all users. TCR sees our APIs as a single user and rate limits our creds. We're hoping that they'll up our rate limits once we start seeing 429 issues

**Comment by jhough07** [comment]
@jmulford-bw please re-review, added a FAQs page

File: campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md (hunk 1 above)
Comment (jmulford-bw) [reply campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md:22]: aaah. Ok sounds good

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**